.gitignore

```diff
@@ -69,3 +69,7 @@ prime/
 *.snap
 *.snap-build
 *_source.tar.bz2
+
+
+# Backup files (i.e. from Vim)
```

Comment:
This should belong in your global git ignore file, or at least create a new PR for it as it is unrelated to the current PR.